.ci/atime/tests.R

```diff
@@ -104,5 +104,17 @@ test.list <- list(
     expr = quote(data.table:::`[.data.table`(dt_mod, , N := .N, by = g)),    
     Before = "be2f72e6f5c90622fe72e1c315ca05769a9dc854", # Parent of the regression causing commit (https://github.com/Rdatatable/data.table/commit/e793f53466d99f86e70fc2611b708ae8c601a451) in the PR that introduced the issue (https://github.com/Rdatatable/data.table/pull/4491/commits)
     Regression = "e793f53466d99f86e70fc2611b708ae8c601a451", # Commit responsible for regression in the PR that introduced the issue (https://github.com/Rdatatable/data.table/pull/4491/commits)
-    Fixed = "58409197426ced4714af842650b0cc3b9e2cb842") # Last commit in the PR that fixed the regression (https://github.com/Rdatatable/data.table/pull/5463/commits)    
+    Fixed = "58409197426ced4714af842650b0cc3b9e2cb842"), # Last commit in the PR that fixed the regression (https://github.com/Rdatatable/data.table/pull/5463/commits)
+
+  # Issue reported in: https://github.com/Rdatatable/data.table/issues/5426
+  # To be fixed in: https://github.com/Rdatatable/data.table/pull/5427
+  "Test performance improvement implemented in #5427" = list(
```

Comment:
This looks like a reasonable new test, but now I notice that all of the test names start with Test, and some include the keyword performance, both of which could be applied to any performance test, so to reduce repetition, can you please remove those keywords from the test names?